MonoGame.Framework/Graphics/RenderTarget2D.DirectX.cs

```diff
@@ -155,8 +155,14 @@ internal override Resource CreateTexture()
 
             // MSAA RT needs another non-MSAA texture where it is resolved
             if (SampleDescription.Count > 1)
-            {
-                _resolvedTexture = new RenderTarget2D(
+                _resolvedTexture = CreateResolvedTexture();
+
+            return rt;
+        }
+
+        protected RenderTarget2D CreateResolvedTexture()
```

Comment:
This better be internal, because not it's accessible to the developer through a derived class.  
(unfortunately C#  doesn't support a `protected AND internal` modifier)

Additionally this could set directly the `_resolvedTexture` with a check/assert to avoid secondary calls.
_resolvedTexture can be reverted back to `private'.

Reply:
Instead of internal, you could use `private protected` (supported since C# 7.2) which translates exactly to `protected AND internal`

Reply:
What weird dialect of C# doesn't support protected internal? It's been supported for a long time now. The only exception is struct members, because a struct cannot be inherited from.

Reply:
nkast specifically stated that the MonoGame developers like to keep most methods internal and don't let classes outside the assembly derive from them. That's why he mentioned the method should be changed from protected to internal. **protected internal** in C# means protected OR internal. That's the reason for my reply that for methods like these, private protected could be used, but I was not sure if these new language features were already used at the time.
Doesn't matter anymore, it was changed to internal (in line with many of the methods in MonoGame).

Reply:
@harmwind Ah, I see what you're getting at now. Seems I misread @nkast's reply